Detectors/CTF/workflow/src/ctf-writer-workflow.cxx

```diff
@@ -44,8 +45,23 @@ WorkflowSpec defineDataProcessing(ConfigContext const& configcontext)
   long run = 0;
   bool doCTF = true, doDict = false, dictPerDet = false;
   if (!configcontext.helpOnCommandLine()) {
-    std::unique_ptr<o2::parameters::GRPObject> grp(o2::parameters::GRPObject::loadFrom(configcontext.options().get<std::string>("grpfile")));
-    dets = grp->getDetsReadOut(configcontext.options().get<std::string>("onlyDet"), configcontext.options().get<std::string>("skipDet"));
+    bool noGRP = configcontext.options().get<bool>("no-grp");
+    auto onlyDet = configcontext.options().get<std::string>("onlyDet");
+    if (!noGRP) {
+      std::unique_ptr<o2::parameters::GRPObject> grp(o2::parameters::GRPObject::loadFrom(configcontext.options().get<std::string>("grpfile")));
+      dets = grp->getDetsReadOut(onlyDet, configcontext.options().get<std::string>("skipDet"));
+      run = grp->getRun();
+    } else {
```

Comment:
for the moment it is ok, but in general, "without a prior simulation" means real data and in this case the GRP is supposed to be available from the CCDB.

Reply:
Understood, but indeed at the moment when we start from real raw data (taken e.g. at Pt2) we have no GRP, right ? (or do I miss sth here ?)

Reply:
Usually we put a fake one, since other workflows need it. Anyway, it was not an objection, just a comment